apps/collection/behaviours/confirm.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var _ = require('underscore');
+
+function getReason(req) {
+  var reasons = [
+    'which-post-office',
+    'under-age',
+    'non-identity',
+    'others-identity',
+    'passport-family',
+    'passport-lost',
+    'no-brp',
+    'other'
+  ];
+  var reason = {};
+
+  if (_.includes(reasons, req.form.values['reason-radio'])) {
```

Comment:
Cheers. Most of the diff in behaviours => controllers is a straight copy-paste job and changed what *needed* to be changed. There's likely to be a lot of things like this.